FCCee/FullSim/IDEA/IDEA_o1_v03/run_digi_reco.py

```diff
@@ -97,18 +97,16 @@
     OutputLevel = INFO
 )
 
-# digitize drift chamber hits, waiting to merge the digitizer for the new drift chamber
-#from Configurables import DCHsimpleDigitizerExtendedEdm
-#dch_digitizer = DCHsimpleDigitizerExtendedEdm("DCHsimpleDigitizerExtendedEdm",
-#    inputSimHits = "CDCHHits",
-#    outputDigiHits = "CDCHDigis",
-#    outputSimDigiAssociation = "DC_simDigiAssociation",
-#    readoutName = "CDCHHits",
-#    xyResolution = 0.1, # mm
-#    zResolution = 1, # mm
-#    debugMode = True,
-#    OutputLevel = INFO
-#)
+from Configurables import DCHdigi_v01
+dch_digitizer = DCHdigi_v01("DCHdigi",
+    DCH_simhits = ["DCHCollection"],
+    DCH_name = "DCH_v2",
+    fileDataAlg = "DataAlgFORGEANT.root",
+    calculate_dndx = False, # cluster counting disabled (to be validated, see FCC-config#239)
+    create_debug_histograms = True,
+    zResolution_mm = 1, # in mm
```

Comment:
After some discussion with DCH experts, let's set the z resolution to 3 cm. For the record: 1 mm resolution is what is expected after using the stereo measurements but here we are before tracking and the z resolution can only rely on the current division and signal time difference at both ends of the wire.